advanced/is-pattern-1/templates/identity-server-conf-entrypoint.yaml

```diff
@@ -38,9 +38,5 @@ data:
     # copy any artifact changes mounted to artifact_volume
     test -d ${artifact_volume} && [ "$(ls -A ${artifact_volume})" ] && cp -RL ${artifact_volume}/* ${WSO2_SERVER_HOME}/
 
-    # make any node specific configuration changes
```

Comment:
Because, this script is modifying the axis2.xml. Since we have the deployment.toml now, we can reference the hostname from the environment variable `HOST_NAME` and it would be resolved at server startup.

Reply:
@Aaquiff then I think we don't think we need this Kubernetes ConfigMap. The purpose of this ConfigMap was to override the default entrypoint script.

Reply:
@chirangaalwis I will remove this

Reply:
@chirangaalwis removed the config map and command parameter from the deployment manifest.